src/server/getters/get-universes-info.ts

```diff
@@ -0,0 +1,39 @@
+import Augur from "augur.js";
+import * as Knex from "knex";
+import { Address, UniverseInfoRow } from "../../types";
+import { formatBigNumberAsFixed } from "../../utils/format-big-number-as-fixed";
+
+interface ParentUniverseRow {
+  parentUniverse: Address;
+}
+
+export function getUniversesInfo(db: Knex, augur: Augur, universe: Address, account: Address, callback: (err: Error|null, result?: any) => void): void {
+  if (universe == null || account == null) return callback(new Error("Must provide both universe and account"));
+
+  const query = db.select("parentUniverse").from("universes").where("universe", universe).first();
+
+  query.asCallback((err: Error|null, parentUniverseRow: ParentUniverseRow) => {
+    if (err) return callback(err);
+
+    const query = db.select(["universes.universe", "universes.parentUniverse", "balances.balance", "token_supply.supply", db.raw("count(markets.marketId) as numMarkets")]).from("universes")
+      .where("universes.parentUniverse", universe) // Children
+      .orWhere("universes.parentUniverse", parentUniverseRow.parentUniverse) // Siblings
+      .orWhere("universes.universe", universe) // Universe
+      .orWhere("universes.universe", parentUniverseRow.parentUniverse) // Parent
+      .leftJoin("token_supply", "token_supply.token", "universes.reputationToken")
+      .leftJoin("balances", function () {
+        this
+          .on("balances.owner", db.raw('"' + account + '"'))
```

Comment:
Should we be concerned about sql injection here?

Reply:
Probably. I'll look at the recommended way to do this and avoid sql injection